src/compile/selection/point.ts

```diff
@@ -1,11 +1,49 @@
-import {Stream} from 'vega';
+import {Signal, Stream} from 'vega';
 import {stringValue} from 'vega-util';
-import {SelectionCompiler, TUPLE, unitName} from '.';
+import {SelectionCompiler, TUPLE, isTimerSelection, unitName} from '.';
 import {SELECTION_ID} from '../../selection';
 import {vals} from '../../util';
 import {BRUSH} from './interval';
 import {TUPLE_FIELDS} from './project';
 
+export const CURR = '_curr';
+
+const animationSignals = (selectionName: string, scaleName: string): Signal[] => {
```

Comment:
How different will these signals be for when we generalize the implementation (eg to have timer-driven interval signals)? Is it worth extracting them to their own selection module and working them in that way for reusability? Happy to punt on that for now, but it'd be useful to spec things through and externalize them while they're more fresh in your head.

Reply:
i feel like i will have a lot of questions about what changes to make to selections to expose predicates so i think i will opt to punt this from this PR's scope but try to write some thoughts down in a doc